frontend/src/pages/distributedWorkloads/global/workloadStatus/DWWorkloadsTable.tsx

```diff
@@ -76,13 +76,13 @@ export const DWWorkloadsTable: React.FC = () => {
 
   return (
     <Card>
-      <CardTitle>Workloads</CardTitle>
+      <CardTitle>Distributed Workload</CardTitle>
       <CardBody>
         <Table
           enablePagination
           data={workloads.data}
           columns={columns}
-          emptyTableView={<>No workloads match your filters</>}
+          emptyTableView={<>No distributed workload match your filters</>}
```

Comment:
workloads should be plural here

Reply:
Hi @mturley I am not sure about this. I will bring this to Breda and get back to you asap.

Reply:
@mturley I asked Katie, the content designer, for this. She suggested that we use "Distributed workload metrics" for now. She thought we could use the single form to stay consistent with other areas (specifically "Project metrics"), but if we get any pushback we can revisit it. Does it make sense?